### PR TITLE
[RW-7885, PF-1376] Pool config for BigQuery Daily Usage Quota Override

### PR DESCRIPTION
**Pull request by jaycarlton**
Add new pool quota for RBS.

### CODE REVIEW

**Review by yuhuyoyo — APPROVED** [review]

File: src/main/resources/config/dev/pool_schema.yml [hunk 1]
```diff
@@ -10,9 +10,9 @@ poolConfigs:
   - poolId: "workspace_manager_v9"
     size: 500
     resourceConfigName: "workspace_manager_v9"
-  - poolId: "vpc_sc_v6"
+  - poolId: "vpc_sc_v7"
```
Comment (melissachang) [inline src/main/resources/config/dev/pool_schema.yml:13]: There's already a `vpc_sc_v7` in prod, so this should be `vpc_sc_v8`.

https://github.com/DataBiosphere/terra-resource-buffer/blob/master/src/main/resources/config/prod/resource-config/vpc_sc_v7.yml

Comment (jaycarlton) [reply src/main/resources/config/dev/pool_schema.yml:13]: good catch

File: src/main/resources/config/dev/pool_schema.yml [hunk 2]
```diff
@@ -10,9 +10,9 @@ poolConfigs:
   - poolId: "workspace_manager_v9"
     size: 500
     resourceConfigName: "workspace_manager_v9"
-  - poolId: "vpc_sc_v6"
```
Comment (calbach) [inline src/main/resources/config/dev/pool_schema.yml:13]: I believe this broke dev, this removedd the old pool which was still in use

Comment (calbach) [reply src/main/resources/config/dev/pool_schema.yml:13]: Sent https://github.com/DataBiosphere/terra-resource-buffer/pull/223

**Pull request merged**